README.md

```diff
@@ -1,40 +1,386 @@
+- Install the CNPG plugin if it is not already on your `PATH`:
+  ```bash
+  curl -sSL https://get.cnpg.io/install | sudo bash
+  kubectl cnpg version
+  ```
+  > If the installer endpoint is unreachable, download the **latest** release directly (replace `v1.27.1` with the newest tag at <https://github.com/cloudnative-pg/cloudnative-pg/releases>):
+  >
+  > ```bash
+  > VERSION="v1.27.1"
+  > curl -L "https://github.com/cloudnative-pg/cloudnative-pg/releases/download/${VERSION}/kubectl-cnpg_${VERSION}_linux_amd64.tar.gz" -o /tmp/kubectl-cnpg.tar.gz
+  > tar -xzf /tmp/kubectl-cnpg.tar.gz -C /tmp
+  > sudo install -m 0755 /tmp/kubectl-cnpg /usr/local/bin/kubectl-cnpg
+  > kubectl cnpg version
+  > ```
+- Optional but recommended: `kubectx`, `stern`, `kubectl-view-secret` (see the [CNPG Playground README](https://github.com/cloudnative-pg/cnpg-playground#prerequisites) for a complete list).
+- Sufficient local resources for a multi-node Kind cluster (≈8 CPUs / 12 GB RAM) and permission to run port-forwards.
+
+Once the tooling is present, everything else is managed via repository scripts and Helm charts.
+
+---
+
+## ⚡ Setup and Configuration
+
+> Follow these sections in order; each references the authoritative upstream documentation to keep this README concise.
+
+### 1. Bootstrap the CNPG Playground
+
+The upstream documentation provides detailed instructions for prerequisites and networking. Follow the setup instructions here: <https://github.com/cloudnative-pg/cnpg-playground#usage>.
+
+Example commands:
+
+```bash
+git clone https://github.com/cloudnative-pg/cnpg-playground.git
+cd cnpg-playground
+./scripts/setup.sh eu         # creates kind-k8s-eu plus MinIO
+./scripts/info.sh             # displays contexts and access information
+export KUBECONFIG=$PWD/k8s/kube-config.yaml
+kubectl config use-context kind-k8s-eu
+```
+
+### 2. Install CloudNativePG and the sample cluster
+
+With the Kind cluster running, install/update the operator by following the official **CloudNativePG v1.27 Installation & Upgrades** guide (<https://cloudnative-pg.io/documentation/1.27/installation_upgrade/>). The snippets below mirror the documented steps:
+
+```bash
+# Re-export the playground kubeconfig if you opened a new shell
+export KUBECONFIG=/path/to/cnpg-playground/k8s/kube-config.yaml
+kubectl config use-context kind-k8s-eu
+
+# Apply the 1.27.1 operator manifest exactly as documented
+kubectl apply --server-side -f \
+  https://raw.githubusercontent.com/cloudnative-pg/cloudnative-pg/release-1.27/releases/cnpg-1.27.1.yaml
+
+# Alternatively, generate a custom manifest via the kubectl cnpg plugin
+kubectl cnpg install generate --control-plane \
+  | kubectl apply --context kind-k8s-eu -f - --server-side
```

Comment:
I will keep one or the other, but not both, it's confusing my first question was "Why I'm installing the same twice?"